test/inductor/test_cooperative_reductions.py

```diff
@@ -232,6 +237,30 @@ def fn(a, b):
             torch.rand(20, 20, device=GPU_TYPE),
         ]
         self.run_and_check(fn, inps, expect_kernel_count=2)
+    def test_quantile_inductor_cuda_dynamic(self):
+        # Regression test for https://github.com/pytorch/pytorch/issues/163759
+        import torch
+        from torch._inductor import config
+
+        if not torch.cuda.is_available():
```

Comment:
Pretty sure we have a decorator for this you can add on the test itself (look for @skipIfNoCuda or something similar)